main/src/ca/uwaterloo/flix/util/Options.scala

```diff
@@ -51,7 +51,7 @@ object Options {
   /**
     * Default test options.
     */
-  val DefaultTest: Options = Default.copy(core = false, test = true, verbosity = Verbosity.Silent)
+  val DefaultTest: Options = Default.copy(inclusion = LibLevel.All, test = true, verbosity = Verbosity.Silent)
```

Comment:
DefaultTest -> TestWithLibAll

Reply:
I kept the `DefaultTest` since that's the "prototype" for the three other test options `TestWithLib(Nix|Min|All)`. Let me know if you prefer just to get rid of this option.